app/controllers/placements/support/schools/mentors_controller.rb

```diff
@@ -4,8 +4,46 @@ class Placements::Support::Schools::MentorsController < Placements::Support::App
 
   def show; end
 
+  def index; end
+
+  def new
+    @mentor_form = mentor_form
+  end
+
+  def check
+    @mentor_form = mentor_form
+    render :new unless @mentor_form.valid?
+  rescue TeachingRecord::RestClient::HttpError
```

Comment:
nice!

Reply:
If we are rescueing it, it shouldn't get sent to Sentry.... I'lll investigate. 

I don't think we want to capture in Sentry when someone has just typed a trn wrong.

Reply:
`I don't think we want to capture in Sentry when someone has just typed a trn wrong.`
Ah that makes sense, Ignore my comment